packages/mobile/src/identity/contactMapping.ts

```diff
@@ -261,36 +258,43 @@ function* fetchWalletAddresses(e164Number: string) {
   return possibleUserAddresses
 }
 
-// Returns IdentifierLookupResult
-// which is Map of identifier -> (Map of address -> AttestationStat)
-export function* lookupAttestationIdentifiers(ids: string[]) {
+// Returns a list of accounts for the identifier received.
+export function* lookupAttestationIdentifiers(id: string) {
   const contractKit = yield call(getContractKit)
   const attestationsWrapper: AttestationsWrapper = yield call([
     contractKit.contracts,
     contractKit.contracts.getAttestations,
   ])
 
-  return yield call([attestationsWrapper, attestationsWrapper.lookupIdentifiers], ids)
+  return yield call([attestationsWrapper, attestationsWrapper.lookupAccountsForIdentifier], id)
 }
 
 // Deconstruct the lookup result and return
 // any addresess that are considered verified
-export function getAddressesFromLookupResult(
-  lookupResult: IdentifierLookupResult,
-  phoneHash: string
-) {
-  if (!lookupResult || !lookupResult[phoneHash]) {
-    return null
+export function* getAddressesFromLookupResult(lookupResult: string[], phoneHash: string) {
```

Comment:
Can we use a more specific type than `string[]`? Is this still of type `IdentifierLookupResult[]`?

Reply:
It's not, `IdentifierLookupResult` has stats information, this is just a list of addresses. Do we have a type for addresses?

Reply:
We do! In @celo/base